pkg/identity/keystone/authorizer_test_policy.json

```diff
@@ -89,6 +89,18 @@
       "values": ["role1"]
     }]
   },
+  {
+    "resource": {
+      "verbs": ["get"],
+      "resources": ["core_resource"],
+      "version": "",
+      "namespace": "*"
+    },
+    "match": [{
+      "type": "role",
```

Comment:
@dixudx that's designed for keystone authorization webhook, please take a look at https://github.com/kubernetes/cloud-provider-openstack/blob/d22e37ede33bfe0bddeb7a9f57d13b74731105e0/pkg/identity/keystone/authorizer.go#L150

Reply:
Hi! Agree with @dixudx that the syntax should be documented. Otherwise it's hard to get what match types are supported by the authorizer without looking at the code.

Match type "role" means that all users with that role are able to access the resource. In this example we stated that any user who has the role "role1" can "get" "core_resource" entities.
Other possible types are: user, group and project. Take a look: https://github.com/kubernetes/cloud-provider-openstack/blob/master/pkg/identity/keystone/policy.go#L33-L39